hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetCache.java

```diff
@@ -184,6 +186,33 @@ public FsDatasetCache(FsDatasetImpl dataset) throws IOException {
     this.memCacheStats = cacheLoader.initialize(this.getDnConf());
   }
 
+  private boolean isPmemCache() {
```

Comment:
Can we remove this method? We may use cache.isTransientCache() to judge instead.